### ISSUE TITLE
Not working on those 2 pages

### ISSUE THREAD

**Issue by Jamaikoulas** [xd4rker/MinerBlock#26]
https://freebitmine.com/btc/ doesn't recognise the miner at all. 
https://www.kulturmagazine.com/bitcoin-faucet/ finds 4 (!!!!) miners, but they still hash.

**Comment by xd4rker** [comment]
Both of the websites use `https://swiftmining.win/static/js/base.js`. Will update soon, thanks!

**Comment by Jamaikoulas** [comment]
ok, i would appreciate it if you send me a notification when that occurs.

Στις Τρί, 21 Αυγ 2018 στις 6:04 μ.μ., ο/η Ismail <notifications@github.com>
έγραψε:

**Comment by xd4rker** [comment]
Okay, sure.

Meanwhile you can add these to your custom filters on the settings page in case you didn't want to wait for the update :)

```
*://*.swiftmining.win/static/js/base.js*
*://*.swiftmining.win/embed/*
*://*.swiftmining.win/go/*
wss://*.wmtech.website/*
*://*.moonify.io/moonify.min.js*
```

**Comment by Jamaikoulas** [comment]
i did and thanx.
Meanwhile, 2 other pages started using a miner than your miner block cannot
even identify.
https://www.crazybitcoin.net and
https://www.crazylite.net

Sorry if i keep you busy.


Στις Τρί, 21 Αυγ 2018 στις 8:14 μ.μ., ο/η Ismail <notifications@github.com>
έγραψε:

**Comment by xd4rker** [comment]
The first site is using a script from `*://*.zymerget.party/*`. The second one looks down.

Please feel free to let me know if there is more. Thank you!

**Comment by xd4rker** [comment]
@Zoeymae Have you tried re-installing Firefox ?

**Comment by Zoeymae** [comment]
This is closed? No I haven't tried that yet but isn't it a Miner? Why would there be an option to ignore the threat? Not sure this is the right place to ask all this; but also wasn't there a button to "Test If it's Working"?  I don't see one.
Thank you very much
I'm not that familiar with Git Hub so be prepared for me to do or say something **"Wrong"**

**Comment by Zoeymae** [comment]
Thank you for the help. I do think i needed that anyway.

**Comment by Jamaikoulas** [comment]
https://dailyfaucet.us check that page also plz.

Στις Τρί, 28 Αυγ 2018 στις 5:52 μ.μ., ο/η Zoeymae <notifications@github.com>
έγραψε:

**Comment by xd4rker** [comment]
@Jamaikoulas dailyfaucet.us doesn't seem to be mining. Do you get any CPU spikes?

**Comment by Jamaikoulas** [comment]
yeah wrong site, i found the site that was hasing, and added a custom line.
Sorry.

Στις Πέμ, 30 Αυγ 2018 στις 8:00 μ.μ., ο/η Ismail <notifications@github.com>
έγραψε:

**Comment by Jamaikoulas** [comment]
https://bitcoinrevenue.org/index
there is a hidden miner that miner block fails to identify

Στις Πέμ, 30 Αυγ 2018 στις 10:09 μ.μ., ο/η jama jamaikoulas <
jamaikoulas@gmail.com> έγραψε:

**Comment by xd4rker** [comment]
thanks, will check it.
